### ISSUE TITLE
Safari menuItems are not tabbable

### ISSUE THREAD

**Issue by mattbrothers** [carbon-design-system/carbon-website#366]
Brief description
Menu Items without children are skipped when tabbing through the ui-shell left side nav menu items

## Detailed description

> Is this a feature request (new component, new icon), a bug, or a general issue? bug

> What did you expect to happen? What happened instead? What would you like to see changed?

> What browser are you working in? Safari 12.1.2 on macOs Mojave

> What offering/product do you work on? Any pressing ship or release dates we should be aware of? [Design Thinking](https://ibm.com/design/thinking/) 

## Steps to reproduce the issue

1. Tab into the left nav items
2. Try to tab into menu items without dropdowns

**Comment by jnm2377** [comment]
@mattbrothers is this still an issue you're seeing? I am not able to replicate it, or maybe I am not trying to tab into the correct thing, but the left nav items seem to be tab-able for me. Could you share a screen recording of the issue you're having?
